website/content/en/docs/contribution-guidelines/releasing.md

```diff
@@ -4,391 +4,275 @@ linkTitle: Releasing
-- **(Major and minor releases only)** `website/config.toml`: update `version_menu = "Releases"` with the patch-less version string `version_menu = "v1.3"`,
-and add the following lines under `[[params.versions]]` for `master`:
-  ```toml
-  [[params.versions]]
-    version = "v1.3"
-    url = "https://v1-3-x.sdk.operatorframework.io"
-  ```
+Create and merge a new PR for the above commit. You can force-merge your PR to the locked-down `v1.3.x`
+if you have admin access to the operator-sdk repo, or ask an administrator to do so.
+
+### 4. Unlock the `v1.3.x` branch
 
-### 5. Lock down proper branch
-Create and merge a new PR for `release-v1.3.0`. Once this PR is merged, lock down the master or release branch
-to prevent further commits between this and step 7. See [this section](#locking-down-branches) for steps to do so.
+See [this guide](#unlocking-branches) for steps to do so.
 
-### 6. Create a release tag, binaries, and signatures
+### 5. Create and push a release tag
 
-The top-level `release.sh` script will take care of verifying versions in files described in step 3, and tagging and verifying the tag, as well as building binaries and generating signatures by calling `make release`.
+```sh
+make tag RELEASE_VERSION=v1.3.1
+git push upstream v1.3.1
+```
 
-Prerequisites:
-- [`git`][doc-git-default-key] and [`gpg`][doc-gpg-default-key] default PGP keys are set locally.
-- Your PGP key is publicly available in a [public key server](#release-signing).
-- _For macOS users:_ GNU `sed` and `make` which are not installed by default. Install them with
-  ```sh
-  $ brew install gnu-sed make
-  ```
-  then ensure they are present in your `$PATH`.
+### 6. Fast-forward the `latest` branch
 
-Call the script with the only argument being the new SDK version:
+The `latest` branch points to the latest release tag to keep the main website subdomain up-to-date.
+Run the following commands to do so:
 
 ```sh
-$ ./release.sh v1.3.0
+git checkout latest
+git reset --hard tags/v1.3.1
+git push -f upstream latest
 ```
 
-`operator-sdk` release binaries and signatures will be in `dist/`. Both binary and signature file names contain version, architecture,
-and platform information; signature file names correspond to the binary they were generated from suffixed with `.asc`.
-For example, signature file `operator-sdk-v1.3.0-x86_64-apple-darwin.asc` was generated from a binary named `operator-sdk-v1.3.0-x86_64-apple-darwin`.
-To verify binaries and tags, see the [verification section](#verifying-a-release).
+### 7. Post release steps
 
-<!-- TODO: remove when ansible/helm operator repos are created and code removed from this repo -->
-`ansible-operator` and `helm-operator` release binaries and signatures are similarly built for upload so `make run`
-can download them in their respective operator type projects. See [#3327](https://github.com/operator-framework/operator-sdk/issues/3327) for details.
+See the [post-release section](#post-release-steps).
```

Comment:
Explicitly call out the post release steps, *even* if it means duplicating the changes.